ros2_batch_job/packaging.py

```diff
@@ -75,48 +67,46 @@ def build_and_test_and_package(args, job):
             '--build-base', '"%s"' % args.buildspace,
             '--install-base', '"%s"' % args.installspace,
             '--cmake-args', '-DBUILD_TESTING=ON', '--no-warn-unused-cli',
-            '--packages-select', 'ros1_bridge',
             '--event-handlers', 'console_direct+',
-        ] + (['--merge-install'] if not args.isolated else []) +
+        ] + (['--packages-select', overlay_pkgs] if overlay_pkgs else []) +
+            (['--merge-install'] if not args.isolated else []) +
             (
                 ['--cmake-args', '-DCMAKE_BUILD_TYPE=' +
                     args.cmake_build_type]
                 if args.cmake_build_type else []
         ), env=env)
         print('# END SUBSECTION')
 
-        if args.test_bridge:
```

Comment:
Doesn't this need a different condition now? `if args.mixed_ros_overlay_pkgs:`

Reply:
I believe this line should be sufficient:
https://github.com/ros2/ci/blob/87db40efa78e47c9bb23da499cc7e5ad4ade52c7/ros2_batch_job/packaging.py#L90

Reply:
That implies that if you don't pass any names (which is the equivalent of not checking the previous checkbox) that all packages are being tested (which is not matching the behavior where nothing gets tested). Therefore I do think the condition is needed.

Reply:
I see. I just noticed that the packages are not being tested in the packaging job. I will change that accordingly.